text/trace/0000-sampling-propagation.md

```diff
@@ -0,0 +1,126 @@
+# Propagate head trace sampling probability
+
+Propose extending the W3C trace context `traceparent` to convey head trace sampling probability.
+
+## Motivation
+
+The head trace probability is useful in child contexts to be able to
+record the effective sampling probability in child spans.  This is
+documented in [OTEP 148](TODO), which establishes semantic conventions
+for conveying the adjusted count of a span via attributes recorded
+with the span.  When a sampling decision is based on the parent's
+context, the effective sampling probability, which determines the
+child's adjusted count, cannot be recorded without propagating it
+through the context.
+
+We propose to propagate the trace sampling probability that is in
+effect alongside the [W3C
+sampled](https://www.w3.org/TR/trace-context/#sampled-flag) flag by
+extending the `traceparent`.
+
+## Explanation
+
+To limit the cost of this extension, to ensure that it is widely
+supported, and for statistical reasons documented below, we propose to
+limit head tracing probability to powers of two.  This limits the
+available head sampling probabilities to 1/2, 1/4, 1/8, and so on, and
+we can compactly encode these probabilities as small integers using
+the negative base-2 logarithm of the effective probability.
+
+For example, the value 2 corresponds with 1-in-4 sampling, the value
+10 corresponds with 1-in-1024 sampling.
+
+Wheres the [version-0 W3C trace context `traceparent`
+header](https://www.w3.org/TR/trace-context/#examples-of-http-traceparent-headers)
+is a concatenation of four fields,
+
+```
+traceparent: (version)-(trace_id)-(span_id)-(flags)
+```
+
+This proposal would upgrade `traceparent` to version 1 with a new
+field named `log-count`,
+
+```
+traceparent: (version)-(trace_id)-(span_id)-(flags)-(log-count)
+```
+
+where `log-count` is the encoded negative base-2 logarithm of
+sampling probability, which is the base-2 logarithm of the adjusted
+count of a child span created in this context (i.e., the logarithm of
+the effective count, thus "log-count").  To compute the adjusted count
+of a child span created in this context, use `2^log-count`.  A
+log-count of `0` corresponds with `(2^0)=1`, thus 0 conveys a context
+with probability 1.
+
+The sampling probability of a context is independent from whether it
+is sampled.  We consider it [useful to convey sampling probability
+even when unsampled]() as it can be used to estimate the potential
```

Comment:
Link?

Reply:
Dappers inflationary sampler is described in OTEP 148 https://github.com/open-telemetry/oteps/blob/88d38b5e82e54317c692a62e730e7aef030dfac2/text/trace/0148-sampling-probability.md#dappers-inflationary-sampler